packages/client/components/DropdownMenuToggle.tsx

```diff
@@ -28,7 +28,7 @@ interface InputStyleProps {
 const InputBlock = styled('div')<InputStyleProps>(
   ({disabled, size}) => ({
     ...ui.fieldBaseStyles,
-    ...ui.fieldSizeStyles[size],
+    ...ui.fieldSizeStyles[size as keyof typeof ui.fieldSizeStyles],
```

Comment:
you're right, but it leaves me confused.
why is using a string to access a constant object OK here? shouldn't size be required to be small, medium, large? if string "foo" got passed in, it would yield undefined...